Packet++/header/HttpLayer.h

```diff
@@ -207,201 +207,217 @@ namespace pcpp
-			Http423Locked,
+			Http423Locked = 423,
 			/** 424 Failed Dependency */
-			Http424FailedDependency,
+			Http424FailedDependency = 424,
+			/** 425 Too Early */
+			Http425TooEarly = 425,
 			/** 426 Upgrade Required */
-			Http426UpgradeRequired,
+			Http426UpgradeRequired = 426,
+			/** 427 Unassigned */
 			/** 428 Precondition Required */
-			Http428PreconditionRequired,
+			Http428PreconditionRequired = 428,
 			/** 429 Too Many Requests */
-			Http429TooManyRequests,
+			Http429TooManyRequests = 429,
+			/** 430 Unassigned */
 			/** 431 Request Header Fields Too Large */
-			Http431RequestHeaderFieldsTooLarge,
-			/** 440 Login Timeout */
-			Http440LoginTimeout,
-			/** 444 No Response */
-			Http444NoResponse,
-			/** 449 Retry With */
-			Http449RetryWith,
-			/** 450 Blocked by Windows Parental Controls */
-			Http450BlockedByWindowsParentalControls,
-			/** 451 (various messages) */
-			Http451,
-			/** 494 Request Header Too Large */
-			Http494RequestHeaderTooLarge,
-			/** 495 Cert Error */
-			Http495CertError,
-			/** 496 No Cert */
-			Http496NoCert,
-			/** 497 HTTP to HTTPS */
-			Http497HTTPtoHTTPS,
-			/** 498 Token expired/invalid */
-			Http498TokenExpiredInvalid,
-			/** 499 (various messages) */
-			Http499,
+			Http431RequestHeaderFieldsTooLarge = 431,
+			/** 432-499 unassigned **/
+			
 			/** 500 Internal Server Error */
-			Http500InternalServerError,
+			Http500InternalServerError = 500,
 			/** 501 Not Implemented */
-			Http501NotImplemented,
+			Http501NotImplemented = 501,
 			/** 502 Bad Gateway */
-			Http502BadGateway,
+			Http502BadGateway = 502,
 			/** 503 Service Unavailable */
-			Http503ServiceUnavailable,
+			Http503ServiceUnavailable = 503,
 			/** 504 Gateway Timeout */
-			Http504GatewayTimeout,
+			Http504GatewayTimeout = 504,
 			/** 505 HTTP Version Not Supported */
-			Http505HTTPVersionNotSupported,
+			Http505HTTPVersionNotSupported = 505,
 			/** 506 Variant Also Negotiates */
-			Http506VariantAlsoNegotiates,
+			Http506VariantAlsoNegotiates = 506,
 			/** 507 Insufficient Storage */
-			Http507InsufficientStorage,
+			Http507InsufficientStorage = 507,
 			/** 508 Loop Detected */
-			Http508LoopDetected,
+			Http508LoopDetected = 508,
 			/** 509 Bandwidth Limit Exceeded */
-			Http509BandwidthLimitExceeded,
+			Http509BandwidthLimitExceeded = 509,
 			/** 510 Not Extended */
-			Http510NotExtended,
+			Http510NotExtended = 510,
 			/** 511 Network Authentication Required */
-			Http511NetworkAuthenticationRequired,
-			/** 520 Origin Error */
-			Http520OriginError,
-			/** 521 Web server is down */
-			Http521WebServerIsDown,
-			/** 522 Connection timed out */
-			Http522ConnectionTimedOut,
-			/** 523 Proxy Declined Request */
-			Http523ProxyDeclinedRequest,
-			/** 524 A timeout occurred */
-			Http524aTimeoutOccurred,
-			/** 598 Network read timeout error */
-			Http598NetworkReadTimeoutError,
-			/** 599 Network connect timeout error */
-			Http599NetworkConnectTimeoutError,
```

Comment:
Why did we remove these values?

Reply:
answered above, let's discuss in the above thread